src/js/flags/util.ts

```diff
@@ -0,0 +1,55 @@
+import { Flag } from "./model";
```

Comment:
I'm not really a fan of naming things "util".  It's super generic and these files tend to grow into grab bags with a bunch of random junk in it (see the two other util files, yuck).  It seems to me that these would be at home in some other sort of higher-level structure - maybe the Flagset class?  Also, I'm just not clear on what they're used for, so bringing in a bit more of the big picture might help.

Reply:
There was only intended to be a small subset of functionality here related to any of the FlagSections established in this folder's dataset. I moved that here since it was more focused on getting a piece of data for conflicts than actually orchestrating the validation of flags.